linkcovery/cli/data.py

```
@@ -33,7 +39,14 @@ def export(
 def import_data(
     file_path: Path = typer.Argument(..., help="File to import (JSON or HTML)"),
```

Ruff (0.15.0)

[warning] Do not perform function call `typer.Argument` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)